### PR TITLE
Task #210084 Change the camp session list UI

### PR DESCRIPTION
**Pull request by chaitanyakole**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [ ] Code is formatted as per format decided
* [ ] Updated acceptance criteria in tracker
* [ ] Updated test cases in test-cases-tracker
https://tracker.tekdi.net/issues/210084

image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by manojLondhe** [comment]
@coderabbitai review

**Comment by manojLondhe** [comment]
@coderabbitai review

File: apps/front-end/src/pages/front-end/Camp/CampDashboard.js [hunk 1]
```diff
@@ -276,13 +276,13 @@ export default function CampDashboard({ footerLinks }) {
...
                     onPress={() => {
                       navigate(`/camps/${campId}/campexecution`);
                     }}
                   >
                     {t("CAMP_EXECUTION")}
-                  </FrontEndTypo.Primarybutton>
+                  </FrontEndTypo.Primarybutton> */}
                 </React.Fragment>
               )}
             </VStack>
```
[the remark this answers is not in the document]
Comment (manojLondhe) [reply apps/front-end/src/pages/front-end/Camp/CampDashboard.js:288]: ignore this if needed

File: apps/front-end/src/pages/admin/camps/CampSessionList.js [hunk 2]
```diff
@@ -87,23 +173,34 @@ export default function CampSessionList({ footerLinks }) {
...
-              <HStack alignItems={"center"} justifyContent={"space-between"}>
-                <HStack alignItems={"center"} space={2}>
+              <HStack justifyContent={"space-between"}>
+                <HStack
+                  space="4"
+                  alignItems={"center"}
+                  // justifyContent={"space-between"}
+                >
                   {item?.session_tracks?.[0]?.status === "complete" && (
                     <CheckCircleIcon color="greenIconColor" size="24px" />
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/camps/CampSessionList.js:205]: use icons from `<IconByName name="CheckCircleIcon" _icon={{color:"greenIconColor",size:"24px"}} />` component

File: apps/front-end/src/pages/admin/camps/CampSessionList.js [hunk 3]
```diff
@@ -114,19 +211,185 @@ export default function CampSessionList({ footerLinks }) {
...
+                onPress={() => handleStartSession(sessionDetails?.id)}
+              >
+                {t("SESSION_STARTED")}
+              </FrontEndTypo.DefaultButton>
+            ) : (
+              <VStack space={4}>
+                {handleCompleteButton() && (
+                  <FrontEndTypo.DefaultButton
+                    borderWidth="0"
+                    background={"#FF0000"}
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/camps/CampSessionList.js:271]: use theme color.

File: apps/front-end/src/pages/admin/camps/CampSessionList.js [hunk 4]
```diff
@@ -114,19 +211,185 @@ export default function CampSessionList({ footerLinks }) {
...
+            <FrontEndTypo.H3
+              textAlign={"Center"}
+              color="textMaroonColor.400"
+              bold
+            >
+              {t("LESSON")} {sessionDetails?.ordering}
+            </FrontEndTypo.H3>
+          </Modal.Header>
+          <Modal.Body p="6">
+            {!["incomplete", "complete"].includes(
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/camps/CampSessionList.js:247]: Please refactor this modal body code.

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**